site/pages/about-us/principles-slice/style.css

```diff
@@ -0,0 +1,55 @@
+@value mediumScreen, largeScreen, maxWidth from "../../../css/_sizes.css";
+@value badgerWhite, badgerRed, badgerBlack from "../../../css/_colors.css";
+
+.principles {
+	background:badgerBlack;
```

Comment:
something is wrong with spacing here, it's a bit inconsistent throughout the file